galsim/phase_psf.py

```diff
@@ -0,0 +1,1017 @@
+# Copyright (c) 2012-2015 by the GalSim developers team on GitHub
+# https://github.com/GalSim-developers
+#
+# This file is part of GalSim: The modular galaxy image simulation toolkit.
+# https://github.com/GalSim-developers/GalSim
+#
+# GalSim is free software: redistribution and use in source and binary forms,
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions, and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions, and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+#
+"""@file phase_psf.py
+Utilities for creating PSFs from phase screens.  Essentially evaluates the Fourier optics
+diffraction equation:
+
+PSF(x, y) = int( |FT(aperture(u, v) * exp(i * phase(u, v, x, y, t)))|^2, dt)
+
+where x, y are focal plane coordinates and u, v are pupil plane coordinates
+
+The main classes of note are:
+
+FrozenAtmosphericScreen
+  Class implementing phase(u, v, x, y, t) for frozen flow von Karman type turbulence
+
+ARAtmosphericScreen
```

Comment:
>Or, do we really need these separate versions? 

I combined ARAtmosphericScreen with FrozenAtmosphericScreen to make plain AtmosphericScreen.  I changed the implementation of non-frozen-flow screens a bit too, so that they're now more like frozen-flow screens.  Only downside is that the former unit test for ARAtmosphericScreen(alpha=1) == FrozenAtmosphericScreen became trivial, so I removed it.  Code looks much cleaner though now.  Thanks for the suggestion.